comp/metadata/inventorysigning/README.md

```diff
@@ -0,0 +1,42 @@
+# Inventory Signing Payload
+
+This component populates the Linux Packages signing information in the `inventories` product in DataDog. They fill the `host_signing` table.
+
+This is enabled by default but can be turned off using `inventories_enabled` config.
+
+The payload is sent every 12h (see `inventories_max_interval` in the config) or whenever it's updated with at most 1
+update every 5 minutes (see `inventories_min_interval`).
```

Comment:
@hush-hush the template config does not contain anything for `inventories_min/max_interval`, should we put something?

Reply:
Until now we didn't had a reason to add them to the configuration template since the defaults are good enough for users (but we have them configurable just in case a use case appears).

Unless you think, for your feature, we need to advertise them to user we can keep them out of the config template.